### ISSUE TITLE
feat: Support "empty" match arm for unit variants

### ISSUE THREAD

**Issue by yuvalsw** [starkware-libs/cairo#3453]
For example, for 
```
enum MyEnum {
   A: (),
   B: usize
}
```
We should support:
```
fn foo(e: MyEnum) {
   match e {
      A => {...}, // Currently we need "A(_) => " or "A(()) => "
      B(x) => {...}
   }
}
```

**Comment by yuvalsw** [comment]
Note that https://github.com/starkware-libs/cairo/issues/3452 is similar/relevant.

**Comment by neotheprogramist** [comment]
I will take that

**Comment by neotheprogramist** [comment]
Fixed by #3569
